source/plugin-settings/obox-storage-plugin.rst

```diff
@@ -137,3 +137,21 @@ There are no migrated POP3 UIDLs. Don't try to look them up in any situation. No
 
 By default doing changes to folders (e.g. creating or renaming) uploads changes immediately to object storage. If this setting is enabled, the upload happens sometimes later (within metacache_upload_interval).
 
+.. _plugin-obox-storage-settting_obox_allow_copy_between_users:
+
+``obox_allow_copy_between_users``
+---------------------------------
+
+.. versionadded:: v2.3.11
+
+If set to yes this setting enables sharing mail-objects in storage between
+users. If enabled it can reduce the storage usage, for example mails
+with multiple recipients, will be only stored once, this also affects expunged
+mails. The deduplication is currently only supported with ``obox_fs`` using
+fs-dictmap. Enabling this setting requires a change to the map configuration see
```

Comment:
I don't think this is true? It probably works with s3/etc as well, since it calls fs_copy() for them also? It just didn't require any special driver code. Would be a good idea to test of course. :)